From 1111111111111111111111111111111111111111 Mon Sep 17 00:00:00 2001
From: Krzesimir Nowak <qdlacz@gmail.com>
Date: Mon, 9 Jul 2018 22:00:01 +0200
Subject: [PATCH 4/4] Drop stuff, add C

---
 foo | 24 ++----------------------
 1 file changed, 2 insertions(+), 22 deletions(-)

diff --git a/foo b/foo
index 1111111..2222222 100644
--- a/foo
+++ b/foo
@@ -1,19 +1,6 @@
 1
 2
 3
-4
-5
-B
-B
-B
-B
-B
-B
-B
-6
-7
-8
-9
 10
 11
 12
@@ -54,9 +41,6 @@
 47
 48
 49
-50
-51
-52
 53
 54
 55
@@ -69,11 +53,6 @@
 62
 63
 64
-65
-66
-67
-68
-69
 70
 71
 72
@@ -90,7 +69,8 @@
 83
 84
 85
-A
+C
+C
 89
 90
 91
-- 
0.0.0

