From 1111111111111111111111111111111111111111 Mon Sep 17 00:00:00 2001
From: Krzesimir Nowak <qdlacz@gmail.com>
Date: Sat, 24 Feb 2018 20:37:55 +0100
Subject: [PATCH 2/3] Add "b"

---
 foo | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/foo b/foo
index 1111111..2222222 100644
--- a/foo
+++ b/foo
@@ -1,19 +1,25 @@
 zz
 aa
+bb
 zz
 zz
 aa
+bb
 zz
 zz
+bb
 aa
 zz
 zz
+bb
 aa
 zz
 zz
 aa
+bb
 zz
 zz
+bb
 aa
 zz
 zz
-- 
0.0.0

